spec/models/participant_declaration_spec.rb

```diff
@@ -16,11 +16,14 @@
         create(:participant_declaration,
                :with_profile_type,
                cpd_lead_provider: call_off_contract.lead_provider.cpd_lead_provider,
-               profile_type: profile_type)
+               profile_type: profile_type,
+               course_identifier: course_identifier)
       end
 
       context "for mentor was created" do
         let(:profile_type) { :mentor_profile }
+        let(:course_identifier) { "ecf-mentor" }
+        let(:course) { :ecf_mentor }
```

Comment:
Is `course` used? Doesn't look like it.

Reply:
Don't think it is in this one. I'll pull it.